### PR TITLE
Lib updates and QG change

### PR DESCRIPTION
**Pull request by adkumar1**

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ How to install application using helm:  <br />
     helm install ReleaseName ChartName
     
     a.) Add helm repository in tractusx:
-           helm repo add sd-factory https://eclipse-tractusx.github.io/charts/dev
+           helm repo add sd-factory https://eclipse-tractusx.github.io/charts/tractusx-dev
```
Comment (almadigabor) [inline INSTALL.md:11]: Is this working? Have you tested it? Naming the repo `sd-factory` and then on line [13](https://github.com/eclipse-tractusx/sd-factory/pull/38/files#diff-09b140a43ebfdd8dbec31ce72cafffd15164d2860fd390692a030bcb932b54a0R13) searching repo with name `tractusx-dev` seems wrong to me.

File: DEPENDENCIES [hunk 2]
```diff
@@ -11,7 +11,7 @@ maven/mavencentral/com.fasterxml.jackson.datatype/jackson-datatype-jdk8/2.14.2,
 maven/mavencentral/com.fasterxml.jackson.datatype/jackson-datatype-jsr310/2.14.2, Apache-2.0, approved, #4699
 maven/mavencentral/com.fasterxml.jackson.jaxrs/jackson-jaxrs-base/2.14.2, Apache-2.0, approved, clearlydefined
 maven/mavencentral/com.fasterxml.jackson.jaxrs/jackson-jaxrs-json-provider/2.14.2, Apache-2.0, restricted, clearlydefined
```
Comment (almadigabor) [inline DEPENDENCIES:13]: Restricted libraries need an Eclipse IP review request to be opened with the [Dash-tool](https://github.com/eclipse/dash-licenses/blob/master/README.md). Please follow the [description ](https://eclipse-tractusx.github.io/docs/release/trg-0/trg-7-04) and update the file with the request ID. Then we can merge. Then after all the dependencies are approved, can the product be part of the release.

File: DEPENDENCIES [hunk 3]
```diff
@@ -23,50 +23,41 @@ maven/mavencentral/com.github.jnr/jnr-ffi/2.0.5, Apache-2.0, approved, CQ12035
...
 maven/mavencentral/com.github.multiformats/java-multibase/v1.1.0, MIT AND BSD-3-Clause AND EPL-1.0 AND Apache-2.0, approved, #4095
 maven/mavencentral/com.github.stephenc.jcip/jcip-annotations/1.0-1, Apache-2.0, approved, CQ21949
-maven/mavencentral/com.google.code.findbugs/jsr305/3.0.2, Apache-2.0, approved, #20
-maven/mavencentral/com.google.errorprone/error_prone_annotations/2.3.4, Apache-2.0, approved, #807
-maven/mavencentral/com.google.guava/failureaccess/1.0.1, Apache-2.0, approved, CQ22654
-maven/mavencentral/com.google.guava/guava/28.2-android, Apache-2.0 AND LicenseRef-Public-Domain, approved, CQ22437
-maven/mavencentral/com.google.guava/listenablefuture/9999.0-empty-to-avoid-conflict-with-guava, Apache-2.0, approved, CQ22657
-maven/mavencentral/com.google.j2objc/j2objc-annotations/1.3, Apache-2.0, approved, CQ21195
-maven/mavencentral/com.google.protobuf/protobuf-java/3.6.1, BSD-3-Clause, approved, clearlydefined
+maven/mavencentral/com.google.protobuf/protobuf-java/3.22.3, , restricted, clearlydefined
```
Comment (almadigabor) [inline DEPENDENCIES:26]: Restricted libraries need an Eclipse IP review request to be opened with the [Dash-tool](https://github.com/eclipse/dash-licenses/blob/master/README.md). Please follow the [description ](https://eclipse-tractusx.github.io/docs/release/trg-0/trg-7-04) and update the file with the request ID. Then we can merge. Then after all the dependencies are approved, can the product be part of the release.

**Review by almadigabor — CHANGES_REQUESTED** [review]
See my comments.

File: INSTALL.md (hunk 1 above)
Comment (adkumar1) [reply INSTALL.md]: This has been fixed

File: DEPENDENCIES (hunk 3 above)
Comment (adkumar1) [reply DEPENDENCIES:26]: @dvasunin : Please check

Comment (adkumar1) [reply DEPENDENCIES:26]: These have been fixed [almadigabor](https://github.com/almadigabor) : please approve

**Comment by almadigabor** [comment]
Please track these IP review requests for you dependencies:

https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8370 
https://gitlab.eclipse.org/eclipsefdn/emo-team/iplab/-/issues/8371

**Review by almadigabor — APPROVED** [review]

**Pull request merged**